### PR TITLE
fix cv dataloader bug

### PR DESCRIPTION
**Pull request by rentainhe**
## TODO
- [x] Fix Imagenet dataloader build function bug

### CODE REVIEW

**Review by CPFLAME — APPROVED** [review]

File: libai/data/build.py [hunk 1]
```diff
@@ -195,23 +196,22 @@ def build_image_train_loader(
...
+
     if sampler is None:
-        # TODO: initialize test_sampler
         sampler = SingleRoundSampler(
             dataset=dataset,
             micro_batch_size=test_batch_size,
             shuffle=False,
             data_parallel_rank=dist.get_data_parallel_rank(),
             data_parallel_size=dist.get_data_parallel_size(),
+            seed=seed,
```
Comment (L1aoXingyu) [inline libai/data/build.py:209]: 我才注意到这个 singleRoundSampler 也需要传 seed，我在想这个 sampler 只会用在 test 的过程中，要不要去掉这些用不到的参数，比如 shuffle，drop_last 之类的
